setup.cfg

```diff
@@ -10,7 +10,7 @@ all_files = 1
 [flake8]
 # classes can be lowercase, arguments and variables can be uppercase
 # whenever it makes the code more readable.
-ignore = N806, N802, N801, N803
+ignore = W504, N806, N802, N801, N803
```

Comment:
`flake8` raised a lot of warnings (not locally, only on Travis) without this change.